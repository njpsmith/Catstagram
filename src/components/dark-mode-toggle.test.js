import { shallow, mount, render } from 'enzyme';
import React from 'react';
import { DarkModeToggle } from './dark-mode-toggle.component';

it('expect to render DarkModeToggle component', () => {
  expect(shallow(<DarkModeToggle />).length).toEqual(1);
});

it('tests DarkModeToggle initially', () => {
  const wrapper = mount(<DarkModeToggle />);
  expect(wrapper).toMatchSnapshot();
});

it('tests DarkModeToggle after toggling', () => {
  const wrapper = mount(<DarkModeToggle />);
  const darkModeToggleSwitch = wrapper.find('#dark-mode-toggle-switch').first();

  darkModeToggleSwitch.simulate('click');

  expect(wrapper).toMatchSnapshot();
});
