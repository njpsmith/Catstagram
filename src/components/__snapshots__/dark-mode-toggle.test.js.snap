// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests DarkModeToggle after toggling 1`] = `ReactWrapper {}`;

exports[`tests DarkModeToggle initially 1`] = `ReactWrapper {}`;
